decred/tests/unit/dcr/test_txscript.py

```diff
@@ -3990,3 +3990,27 @@ def test_put_var_int():
         assert (
             res == test["want"]
         ), f'wanted {test["want"].hex()} but got {res.hex()} for test {test["name"]}'
+
+
+def test_verify_sig():
+    """
+    pub (PublicKey): The public key.
+    inHash (byte-like): The thing being signed.
+    r (int): The R-parameter of the ECDSA signature.
+    s (int): The S-parameter of the ECDSA signature.
+    """
+    key = parseShortForm(
+        "0xea 0xf0 0x2c 0xa3 0x48 0xc5 0x24 0xe6 "
+        "0x39 0x26 0x55 0xba 0x4d 0x29 0x60 0x3c "
+        "0xd1 0xa7 0x34 0x7d 0x9d 0x65 0xcf 0xe9 "
+        "0x3c 0xe1 0xeb 0xff 0xdc 0xa2 0x26 0x94 "
+    )
+
+    priv = crypto.privKeyFromBytes(key)
+    pub = priv.pub
+
+    inHash = parseShortForm("0x00 0x01 0x02 0x03 0x04 0x05 0x06 0x07 0x08 0x09")
+    sig = txscript.signRFC6979(priv.key, inHash)
+    assert txscript.verifySig(pub, inHash, sig.r, sig.s)
+
```

Comment:
There are three negative paths to check in `verifySig`.

1. `r <= 0 or s <= 0`
2. `r >= Curve.N or s >= Curve.N`
3. invalid signature. Can just bump `s += 1`  or something to accomplish this.